input/fsh/profiles/ROROrganization.fsh

```diff
@@ -120,24 +120,43 @@ ou nom de l'OI : Nom de l'organisation interne"
 * address.extension contains
     RORCalculatedDistance named ror-calculated-distance 0..1
 * address.extension[ror-calculated-distance] ^short = "Eloignement calculé"
+
+* address.line.extension ^slicing.discriminator[0].type = #value
+* address.line.extension ^slicing.discriminator[=].path = "url"
+* address.line.extension ^slicing.rules = #open
 * address.line.extension contains
     iso21090-ADXP-careOf named careOf 0..1 and
     iso21090-ADXP-additionalLocator named additionalLocator 0..1 and
     iso21090-ADXP-houseNumber named houseNumber 0..1 and
     iso21090-ADXP-buildingNumberSuffix named buildingNumberSuffix 0..1 and
     iso21090-ADXP-streetNameType named streetNameType 0..1 and
-    iso21090-ADXP-streetNameBase named streetNameBase 0..1 and
-    iso21090-ADXP-streetNameBase named streetNameHamlet 0..1 and
+    iso21090-ADXP-streetNameBase named streetName 0..2 and
     iso21090-ADXP-postBox named postalBox 0..1
+
 * address.line.extension[careOf] ^short = "pointRemise (Adresse)"
+* address.line.extension[careOf].id = "careOf"
 * address.line.extension[additionalLocator] ^short = "complementPointGeographique (Adresse)"
+* address.line.extension[additionalLocator].id = "additionalLocator"
 * address.line.extension[houseNumber] ^short = "numeroVoie(Adresse)"
+* address.line.extension[houseNumber].id = "houseNumber"
 * address.line.extension[buildingNumberSuffix] ^short = "(Adresse)"
+* address.line.extension[buildingNumberSuffix].id = "buildingNumberSuffix"
 * address.line.extension[streetNameType] ^short = "typeVoie (Adresse)"
+* address.line.extension[streetNameType].id = "streetNameType"
 * address.line.extension[streetNameType].valueString from $JDV-J219-TypeVoie-ROR (required)
-* address.line.extension[streetNameBase] ^short = "libelleVoie (Adresse)"
-* address.line.extension[streetNameHamlet] ^short = "lieuDit (Adresse)"
 * address.line.extension[postalBox] ^short = "mentionDistribution (Adresse)"
+* address.line.extension[postalBox].id = "postalBox"
+
+* address.line.extension[streetName] ^slicing.discriminator[+].type = #value
```

Comment:
il y a un warning sushi comme pour la PR https://github.com/ansforge/IG-fhir-repertoire-offre-ressources-sante/pull/132
warn  Organization.address.line.extension[streetName]: An element with a slice name should not define its own slicing. Instead, append additional discriminators to the original slicing on the base element.
  File: /home/runner/work/IG-fhir-repertoire-offre-ressources-sante/IG-fhir-repertoire-offre-ressources-sante/input/fsh/profiles/ROROrganization.fsh
  Line: 1 - 211